packages/Tabs/src/components/List/List.js

```diff
@@ -1,48 +1,56 @@
 import React from "react";
 import PropTypes from "prop-types";
+
 import TabsContext from "../../TabsContext";
 import * as sc from "./List.styles";
 
-const propTypes = {
-  /** Descriptive a11y text for assistive technologies. By default, text from children node will be used. */
-  a11yText: PropTypes.string,
-  children: PropTypes.node.isRequired,
-  height: PropTypes.number,
-};
-
-const defaultProps = {
-  a11yText: null,
-  height: null,
-};
-
 export default function List(props) {
   const context = React.useContext(TabsContext);
 
-  const { a11yText, children, height, ...moreProps } = props;
-  const { activeIndex, kind, currentFocusIndex, onKeyDown, onClickTab, setTabListRef } = context;
+  const { children, ...moreProps } = props;
+  const {
+    activeIndex,
+    kind,
+    currentFocusIndex,
+    hasInsetFocusStyle,
+    hasTruncation,
+    tabHeight,
+    isVertical,
+    onKeyDown,
+    onClickTab,
+    setTabListRef,
+    size,
+  } = context;
 
   const childrenWithProps = React.Children.map(children, (tab, index) => {
-    const isSelected = activeIndex === index;
+    if (!tab) return;
```

Comment:
I think this could be removed since we're already filtering out `null` TabsList children in `Tabs.js`

Reply:
Whatever we decide for this, we should apply to `Panels` as well to keep things consistent. Right now, looks like the `null` check is done in line 15 in `Panels.js`

Reply:
The null check in `Tabs.js` is just for handling keyboard navigation, not for rendering purposes.  

This line here (and the corresponding one in `Panels.js`) is to allow `null` children by avoiding the attempt to `cloneElement()` on them, which throws an error.